examples/debug/example.py

```diff
@@ -36,20 +30,8 @@ def process(self, key, value):
         except ValueError as ve:
             log.exception(ve)
             return
-        self.store.add(key, str(doubled))
-
-        # TODO: In absence of a punctuate call schedule running:
-        if len(self.store) == 4:
-            self.punctuate()
-
-            self.context.commit()
-
-    def punctuate(self):
-        log.debug('DoubleProcessor::punctuate')
-        for k, v in iter(self.store):
-            log.debug('Forwarding to sink  (%s, %s)', k, v)
-            self.context.forward(k, v)
-        self.store.clear()
+        log.debug(f'Forwarding to sink ({key}, {str(doubled)})')
+        self.context.forward(key, str(doubled))
 
```

Comment:
Not needed, this just multiplies every record by 2, library should do all the boilerplate.